airflow/api_fastapi/core_api/routes/public/task_instances.py

```diff
@@ -234,6 +236,70 @@ def get_task_instance_dependencies(
     return TaskDependencyCollectionResponse.model_validate({"dependencies": deps})
 
 
+@task_instances_router.get(
+    task_instances_prefix + "/{task_id}/tries",
+    responses=create_openapi_http_exception_doc([status.HTTP_404_NOT_FOUND]),
+)
+def get_task_instance_tries(
+    dag_id: str,
+    dag_run_id: str,
+    task_id: str,
+    session: Annotated[Session, Depends(get_session)],
+    map_index: int = -1,
```

Comment:
map_index will show up in the OpenAPI specifications. So, the user might send map_index here.

Reply:
`get_task_instance_tries` will work for non mapped tasks. For mapped tasks which requires map_index, there is a different endpoint `get_mapped_task_instance_tries`.